packages/bundle-size/src/commands/compareReports.test.js

```diff
@@ -0,0 +1,29 @@
+const getRemoteReport = jest.fn();
+const collectLocalReport = jest.fn();
+const compareResultsInReports = jest.fn();
+const cliReporter = jest.fn();
+
+jest.mock('../reporters/cliReporter', () => cliReporter);
+jest.mock('../utils/collectLocalReport', () => collectLocalReport);
+jest.mock('../utils/compareResultsInReports', () => ({ compareResultsInReports }));
+jest.mock('../utils/getRemoteReport', () => getRemoteReport);
+
+const sampleReport = require('../../__fixture__/sampleReport');
+const sampleComparedReport = require('../../__fixture__/sampleComparedReport');
+const { handler } = require('./compareReports');
+
+describe('compareReports', () => {
```

Comment:
we need to come up with some more sophisticated e2e testing for our CLI tools, while this is definitely better than nothing, we are testing too much implementation details via mocking etc.

anyways thanks you!